[CLC-181] Updated Documentation (#241)
hazelcast/hazelcast-commandline-client: docs/modules/ROOT/pages/overview.adoc

{description}

> The Hazelcast CLC is a single binary with no dependencies. Within minutes of installation, you can start to perform common tasks on Viridian and Hazelast clusters.

Reviewer: ```suggestion
The Hazelcast CLC is a single binary with no dependencies. Within minutes of installation, you can start to perform common tasks on {hazelcast-cloud} and Hazelcast clusters.
```

Example use cases for the Hazelcast CLC.

=== Run SQL Queries

Run xref:clc-sql.adoc[SQL] queries against your cluster and display the output as a table or in a variety of text formats, such as JSON and CSV.

=== Manage Viridian Clusters

> Using CLC, you can xref:clc-viridian.adoc[create and manage] Viridian clusters and custom classes on those clusters.

Reviewer: ```suggestion
Using the Hazelcast CLC, you can xref:clc-viridian.adoc[create and manage] {hazelcast-cloud} clusters, and the custom classes on those clusters.
```

> Using the Hazelcast CLC, you can xref:clc-viridian.adoc[create and manage] {hazelcast-cloud} clusters, and the custom classes on those clusters.

Reviewer: ```suggestion
xref:clc-viridian.adoc[Create and manage] {hazelcast-cloud} clusters, and the custom classes on those clusters.
```

Author: Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/241/commits/94fca0906227805df773d012ae8785286a360d6a

=== Create Data Pipelines

> You can xref:clc-job.adoc[create and manage] data pipelines using CLC. Check out the Platform documentation for more information about data pipelines.

Reviewer: ```suggestion
You can xref:clc-job.adoc[create and manage] data pipelines using the Hazelcast CLC. Check out the Platform documentation for more information about data pipelines.
```
It would be good to provide a link to the appropriate page.

Reviewer: Here is the syntax xref:hazelcast:pipelines:overview.adoc[Platform documentation]

Author: Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/241/commits/078ec910da7d1053d167e6eafce0feadee1af0d1

> You can xref:clc-job.adoc[create and manage] data pipelines using the Hazelcast CLC. Check out xref:hazelcast:pipelines:overview.adoc[Platform documentation] for more information about data pipelines.

Reviewer: ```suggestion
You can xref:clc-job.adoc[create and manage] data pipelines using the Hazelcast CLC. Check out the xref:hazelcast:pipelines:overview.adoc[Platform documentation] for more information about data pipelines.
```

> You can xref:clc-job.adoc[create and manage] data pipelines using the Hazelcast CLC. Check out xref:hazelcast:pipelines:overview.adoc[Platform documentation] for more information about data pipelines.

Author: Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/241/commits/94fca0906227805df773d012ae8785286a360d6a
